airflow_clickhouse_plugin/hooks/__init__.py

```diff
@@ -1 +0,0 @@
-from .clickhouse_hook import ClickHouseHook
```

Comment:
Which interface do plugins expose with Airflow v2? How should I import the plugin parts now?

Reply:
I've changed the readme with the new changes on the import.